models/multitask_centernet/demo.py

```diff
@@ -0,0 +1,126 @@
+import os
+import numpy as np
+import cv2
+import argparse
+import onnxruntime
+from tqdm import tqdm
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--model-path", type=str, default="./best.onnx")
+parser.add_argument("--img-path", type=str, default="./sample_ips.txt")
+parser.add_argument("--dst-path", type=str, default="./sample_ops_onnxrt")
+args = parser.parse_args()
+
+
+_CLASS_COLOR_MAP = [
+    (0, 0, 255) , # Person (blue).
+    (255, 0, 0) ,  # Bear (red).
+    (0, 255, 0) ,  # Tree (lime).
+    (255, 0, 255) ,  # Bird (fuchsia).
+    (0, 255, 255) ,  # Sky (aqua).
+    (255, 255, 0) ,  # Cat (yellow).
+]
+
+palette = np.array([[255, 128, 0], [255, 153, 51], [255, 178, 102],
+                    [230, 230, 0], [255, 153, 255], [153, 204, 255],
+                    [255, 102, 255], [255, 51, 255], [102, 178, 255],
+                    [51, 153, 255], [255, 153, 153], [255, 102, 102],
+                    [255, 51, 51], [153, 255, 153], [102, 255, 102],
+                    [51, 255, 51], [0, 255, 0], [0, 0, 255], [255, 0, 0],
+                    [255, 255, 255]])
+
+skeleton = [[16, 14], [14, 12], [17, 15], [15, 13], [12, 13], [6, 12],
+            [7, 13], [6, 7], [6, 8], [7, 9], [8, 10], [9, 11], [2, 3],
+            [1, 2], [1, 3], [2, 4], [3, 5], [4, 6], [5, 7]]
+
+pose_limb_color = palette[[9, 9, 9, 9, 7, 7, 7, 0, 0, 0, 0, 0, 16, 16, 16, 16, 16, 16, 16]]
+pose_kpt_color = palette[[16, 16, 16, 16, 16, 0, 0, 0, 0, 0, 0, 9, 9, 9, 9, 9, 9]]
+radius = 5
+
+def read_img(img_file, img_mean=127.5, img_scale=1/127.5):
+    img = cv2.imread(img_file)[:, :, ::-1]
+    img = cv2.resize(img, (640,640), interpolation=cv2.INTER_LINEAR)
+    img = (img - img_mean) * img_scale
+    img = np.asarray(img, dtype=np.float32)
+    img = np.expand_dims(img,0)
+    img = img.transpose(0,3,1,2)
+    return img
+
+
+def model_inference(model_path=None, input=None):
+    #onnx_model = onnx.load(args.model_path)
+    session = onnxruntime.InferenceSession(model_path, None)
```

Comment:
Please, use `cv2.dnn` for inference. You can check other models in the zoo for examples.